addon/lib/shooter.js

```diff
@@ -198,10 +198,15 @@ const ShotContext = Class({
       }).bind(this)));
     }, this));
     this.interactiveWorker.port.on("textSelection", watchFunction(function (textSelection) {
-      this.activeClipName = this.shot.addClip({
-        createdDate: Date.now(),
-        text: textSelection
-      });
+      if (this.activeClipName){
+        let c = this.shot.getClip(this.activeClipName);
+        c.text = textSelection;
```

Comment:
This can't happen currently, can it?  Since text selection is only on the add-clip menu.  This code wouldn't be correct if, e.g., the active clip was an image clip.

Probably there's a bug currently if you do a text selection then hit the "visible" or something that produces an image clip.